helm/kubernetes-coredns-chart/templates/configmap.yaml

```diff
@@ -24,8 +24,10 @@ data:
         }
         prometheus :{{ .Values.ports.prometheus }}
 
-        {{- range .Values.configmap.proxy }}
-        proxy {{ . }}
+        {{- if .Values.configmap.proxy }}
+        {{- range (.Values.configmap.proxy | trimAll "\n " |  split "\n") }}
+        proxy {{ . | trimAll "- " }}
```

Comment:
If `.Values.configmap.proxy` is a yaml list why you can't just iterate over it?

Reply:
It is a string not a list, hence the logic to transform a multi line string into a list. Feels hackish, but didn’t found a more elegant way to achieve this

Reply:
So where does the `- ` come from?

Reply:
Can you point me to an example value?

Reply:
And finally if this is just multiline problem: isn't it sufficient to use `| quote`?

Reply:
There’s an example of value in this PR, under `data.proxy`.
Yeah right the “-“ is not needed there, and confusing. I’ll remove this.
AFAIK `quote` will only ensure the final string is quoted.

Reply:
I would expect `quote` to also respect whitespaces.

Reply:
Tried it and `quote` does not work in this case, as it produce one big line.

```
$ helm template ./helm/kubernetes-coredns-chart -x templates/configmap.yaml -f coredns-user-values_multiple_trailing_line.yaml
Error: render error in "kubernetes-coredns-chart/templates/configmap.yaml": template: kubernetes-coredns-chart/templates/configmap.yaml:28:45: executing "kubernetes-coredns-chart/templates/configmap.yaml" at <quote>: range can't iterate over "foo.com 1.1.1.1\nbar.com 8.8.8.8\n"
```

Reply:
But why you need to iterate when it's quoted?

Reply:
That was tough :D